pkg/api/operators/apis/experiment/v1alpha2/util.go

```diff
@@ -73,10 +73,34 @@ func (exp *Experiment) IsFailed() bool {
 	return hasCondition(exp, ExperimentFailed)
 }
 
+func (exp *Experiment) IsRunning() bool {
+	return hasCondition(exp, ExperimentRunning)
+}
+
+func (exp *Experiment) IsRestarting() bool {
+	return hasCondition(exp, ExperimentRestarting)
+}
+
 func (exp *Experiment) IsCompleted() bool {
 	return exp.IsSucceeded() || exp.IsFailed()
 }
 
+func (exp *Experiment) GetCurrentConditionType() ExperimentConditionType {
```

Comment:
it will always return `ExperimentCreated`, I don't think you except it

Reply:
@hougangliu We can leave here only `Succeeded` or `Failed` or `Running` or `Restarting` condition. I think, for 1 experiment it can be one of the 4 conditions at the same time, isn't it?

Reply:
I mean `exp.IsCreated()` will always `true`

Reply:
What will happen in that situation:
Experiment ran and after that failed
Experiment restarted
Experiment is running again.
Will `exp.IsRunning()` and `exp.IsFailed()` return true ?

Reply:
My understanding, the `exp.Status.Conditions` is a history of conditions of the experiment. So the exp.IsCreated() will be always `true`.
The last value of `exp.Status.Conditions` is the latest condition.

Reply:
@YujiOshima @hougangliu Ok, thank you! I changed this function to get the latest condition, please , take a look. In the UI, where we monitor experiments, I think, we should show the latest condition.